### PR TITLE
Update translations [pl-PL]

### PR DESCRIPTION
**Pull request by Daxxxis**
**Details of Changes**
Add missing Polish translations

**Validation**
How did you test these changes?
- [x] Built and ran the app

### CODE REVIEW

**Comment by yaichenbaum** [comment]
@Daxxxis before we merge this, can you add your agreement to #7625?

File: src/Files/MultilingualResources/Files.pl-PL.xlf [hunk 1]
```diff
@@ -3193,279 +3192,279 @@ Korzystamy z App Center, aby śledzić wykorzystanie aplikacji, znajdować błę
...
         </trans-unit>
         <trans-unit id="None" translate="yes" xml:space="preserve">
           <source>None</source>
-          <target state="new">None</target>
+          <target state="translated">Brak</target>
         </trans-unit>
         <trans-unit id="HighDpiOption_UseDPIOnLogin" translate="yes" xml:space="preserve">
           <source>On Windows login</source>
-          <target state="new">On Windows login</target>
+          <target state="translated">Przy zalogowaniu</target>
```
Comment (d2dyno1) [inline src/Files/MultilingualResources/Files.pl-PL.xlf:3275]: Przy zalogowaniu do Windows ?

File: src/Files/MultilingualResources/Files.pl-PL.xlf [hunk 2]
```diff
@@ -3193,279 +3192,279 @@ Korzystamy z App Center, aby śledzić wykorzystanie aplikacji, znajdować błę
...
         </trans-unit>
         <trans-unit id="HighDpiOverride_System" translate="yes" xml:space="preserve">
           <source>System</source>
-          <target state="new">System</target>
+          <target state="translated">System</target>
         </trans-unit>
         <trans-unit id="HighDpiOverride_SystemAdvanced" translate="yes" xml:space="preserve">
           <source>System (advanced)</source>
-          <target state="new">System (advanced)</target>
+          <target state="translated">System (ulepszony)</target>
```
Comment (d2dyno1) [inline src/Files/MultilingualResources/Files.pl-PL.xlf:3287]: System (zaawansowane)

File: src/Files/MultilingualResources/Files.pl-PL.xlf [hunk 3]
```diff
@@ -3193,279 +3192,279 @@ Korzystamy z App Center, aby śledzić wykorzystanie aplikacji, znajdować błę
...
-          <target state="new">This setting modifies system files and can have unexpected side effects on your device. The developers take no responsibility in the event an issue occurs as a result. Continuing with this option is an acknowledgment of the risks involved with this action. Please note, uninstalling Files will not undo these changes and may prevent you from opening file explorer unless you turn off this setting before removing Files from your device.</target>
+          <target state="translated">To ustawienie modyfikuje pliki systemu i może wywołać niechciane efekcty uboczne na Twoim urządzeniu. Deweloperzy nie ponoszą odpowiedzialności w przypadku wystąpienia problemów. Kontynuowanie z tą opcją jest potwierdzeniem akceptacji ryzyka związanego z tym działaniem. Pamiętaj, że odinstalowanie aplikacji Files nie cofnie tych zmian i może uniemożliwić otwarcie eksploratora plików, chyba że wyłączysz to ustawienie przed usunięciem apikacji Files z urządzenia.</target>
```
Comment (d2dyno1) [inline src/Files/MultilingualResources/Files.pl-PL.xlf:3391]: niechciane efekcty -> niechciane efekty

File: src/Files/MultilingualResources/Files.pl-PL.xlf (hunk 2 above)
Comment (Daxxxis) [reply src/Files/MultilingualResources/Files.pl-PL.xlf:3287]: image

File: src/Files/MultilingualResources/Files.pl-PL.xlf (hunk 3 above)
Comment (Daxxxis) [reply src/Files/MultilingualResources/Files.pl-PL.xlf:3391]: 👍

File: src/Files/MultilingualResources/Files.pl-PL.xlf (hunk 2 above)
Comment (d2dyno1) [reply src/Files/MultilingualResources/Files.pl-PL.xlf:3287]: :eyes:

**Review by d2dyno1 — APPROVED** [review]

**Pull request merged**